crypto/x509/x509_test.cc

```diff
@@ -2302,6 +2661,78 @@ TEST(X509Test, Ed25519Sign) {
   ASSERT_TRUE(SignatureRoundTrips(md_ctx.get(), pub.get()));
 }
 
+TEST(X509Test, Dilithium3SignVerifyCert) {
+  //generate the dilithium key
```

Comment:
Nit: I know it has been reviewed. But IMO, these comments are mostly redundant since it is obvious what is going on.

Reply:
Now that some other stuff is changing anyway, could fix the indentation.

Reply:
sure, will fix indentation and remove particularly obvious comments.

Reply:
comments consolidate.

what's the issue with indentation? 2 spaces seems consistent with the rest of the file.